src/commands/generateEditorConfig.ts

```diff
@@ -0,0 +1,45 @@
+import {exists, writeFile} from 'fs';
+import {workspace, window} from 'vscode';
+import {Utils} from '../utils';
+
+const supportedProperties = ['indent_style', 'indent_size', 'tab_width'];
```

Comment:
I guess my issue, then, is more with the name of this variable. We support more than these properties, so we should name this something like `generatedProperties` or something.